crs_msgs/srv/CallFreespaceMotion.srv

```diff
@@ -1,4 +1,5 @@
 string target_link
+trajectory_msgs/JointTrajectoryPoint start_position 	# leave empty to use current position
```

Comment:
I would use sensor_msgs/JointState instead of trajectory_msgs/JointTrajectoryPoint, we don't need a time_from_start parameter and the JointState msg definition also includes a vector of strings for joint names which can be nice to make sure all joint positions are correctly applied to the right joints

Reply:
I'll make the change